mmdet/models/roi_heads/mask_heads/fcn_mask_head.py

```diff
@@ -260,7 +262,11 @@ class label c.
             if threshold >= 0:
                 masks = (masks >= threshold).to(dtype=torch.bool)
             else:
-                masks = (masks * 255).to(dtype=torch.bool)
+                # TensorRT backend does not have data type of uint8
```

Comment:
`masks = (masks * 255).to(dtype=torch.bool) ` aims to convert it to bool type. Shall we add a logic to check that? If it is not for exportation, convert it to bool could accelerate the speed.

Reply:
It is in the block of if `torch.onnx.is_in_onnx_export()`, so no need to check.